--- libjxl-0.10.2/third_party/CMakeLists.txt.orig	2024-03-08 09:10:58.000000000 +0100
+++ libjxl-0.10.2/third_party/CMakeLists.txt	2024-04-05 16:53:39.078193231 +0200
@@ -95,15 +95,6 @@ else()
 endif()
 
 # *cms
-if (JPEGXL_ENABLE_SKCMS)
-  if( NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/skcms/skcms.h" )
-    message(FATAL_ERROR "Please run ${PROJECT_SOURCE_DIR}/deps.sh to fetch the "
-            "build dependencies.")
-  endif()
-  include(skcms.cmake)
-  configure_file("${CMAKE_CURRENT_SOURCE_DIR}/skcms/LICENSE"
-                 ${PROJECT_BINARY_DIR}/LICENSE.skcms COPYONLY)
-endif ()
 if (JPEGXL_ENABLE_VIEWERS OR NOT JPEGXL_ENABLE_SKCMS OR JPEGXL_ENABLE_PLUGINS)
   if( NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/lcms/.git" OR JPEGXL_FORCE_SYSTEM_LCMS2 )
     find_package(LCMS2 2.12)
@@ -162,14 +153,3 @@ else()
                    ${PROJECT_BINARY_DIR}/LICENSE.libpng COPYONLY)
   endif()  # JPEGXL_DEP_LICENSE_DIR
 endif()
-
-# sjpeg
-if (JPEGXL_ENABLE_SJPEG)
-  if (NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/sjpeg/CMakeLists.txt")
-    message(FATAL_ERROR "Please run ${PROJECT_SOURCE_DIR}/deps.sh to fetch the "
-            "build dependencies.")
-  endif()
-  include(sjpeg.cmake)
-  configure_file("${CMAKE_CURRENT_SOURCE_DIR}/sjpeg/COPYING"
-                 ${PROJECT_BINARY_DIR}/LICENSE.sjpeg COPYONLY)
-endif ()
