--- libjxl-0.8.1/third_party/CMakeLists.txt.orig	2023-03-05 22:20:51.206401583 +0100
+++ libjxl-0.8.1/third_party/CMakeLists.txt	2023-03-06 19:40:16.610527695 +0100
@@ -100,15 +100,6 @@
 endif()
 
 # *cms
-if (JPEGXL_ENABLE_SKCMS OR JPEGXL_ENABLE_PLUGINS)
-  if( NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/skcms/skcms.h" )
-    message(FATAL_ERROR "Please run ${PROJECT_SOURCE_DIR}/deps.sh to fetch the "
-            "build dependencies.")
-  endif()
-  include(skcms.cmake)
-  configure_file("${CMAKE_CURRENT_SOURCE_DIR}/skcms/LICENSE"
-                 ${PROJECT_BINARY_DIR}/LICENSE.skcms COPYONLY)
-endif ()
 if (JPEGXL_ENABLE_VIEWERS OR NOT JPEGXL_ENABLE_SKCMS)
   if( NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/lcms/.git" OR JPEGXL_FORCE_SYSTEM_LCMS2 )
     find_package(LCMS2 2.13)
@@ -167,14 +158,3 @@
                    ${PROJECT_BINARY_DIR}/LICENSE.libpng COPYONLY)
   endif()  # JPEGXL_DEP_LICENSE_DIR
 endif()
-
-# sjpeg
-if (JPEGXL_ENABLE_SJPEG)
-  if (NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/sjpeg/CMakeLists.txt")
-    message(FATAL_ERROR "Please run ${PROJECT_SOURCE_DIR}/deps.sh to fetch the "
-            "build dependencies.")
-  endif()
-  include(sjpeg.cmake)
-  configure_file("${CMAKE_CURRENT_SOURCE_DIR}/sjpeg/COPYING"
-                 ${PROJECT_BINARY_DIR}/LICENSE.sjpeg COPYONLY)
-endif ()
--- libjxl-0.8.1/lib/jxl.cmake.orig	2023-02-03 13:15:36.000000000 +0100
+++ libjxl-0.8.1/lib/jxl.cmake	2023-03-06 19:41:57.656646948 +0100
@@ -453,17 +453,6 @@
 target_link_libraries(jxl_enc-obj PUBLIC jxl_profiler)
 endif()
 
-#TODO(lode): don't depend on CMS for the core library
-if (JPEGXL_ENABLE_SKCMS)
-  target_include_directories(jxl_enc-obj PRIVATE
-    $<TARGET_PROPERTY:skcms,INCLUDE_DIRECTORIES>
-  )
-else ()
-  target_include_directories(jxl_enc-obj PRIVATE
-    $<TARGET_PROPERTY:lcms2,INCLUDE_DIRECTORIES>
-  )
-endif ()
-
 # Generate version.h
 configure_file("jxl/version.h.in" "include/jxl/version.h")
 
@@ -505,9 +494,6 @@
   $<TARGET_OBJECTS:jxl_enc-obj>
   $<TARGET_OBJECTS:jxl_dec-obj>
 )
-if (JPEGXL_ENABLE_SKCMS AND JPEGXL_BUNDLE_SKCMS)
-  list(APPEND JPEGXL_INTERNAL_OBJECTS $<TARGET_OBJECTS:skcms-obj>)
-endif()
 
 # Private static library. This exposes all the internal functions and is used
 # for tests.
--- libjxl-0.8.1/plugins/gdk-pixbuf/CMakeLists.txt.orig	2023-03-06 19:40:46.967029907 +0100
+++ libjxl-0.8.1/plugins/gdk-pixbuf/CMakeLists.txt	2023-03-06 19:42:39.186421962 +0100
@@ -23,7 +23,7 @@
 
 # Note: This only needs the decoder library, but we don't install the decoder
 # shared library.
-target_link_libraries(pixbufloader-jxl jxl jxl_threads skcms-interface PkgConfig::Gdk-Pixbuf)
+target_link_libraries(pixbufloader-jxl jxl jxl_threads skcms PkgConfig::Gdk-Pixbuf)
 
 execute_process(COMMAND ${PKG_CONFIG_EXECUTABLE} gdk-pixbuf-2.0 --variable gdk_pixbuf_moduledir --define-variable=prefix=${CMAKE_INSTALL_PREFIX} OUTPUT_VARIABLE GDK_PIXBUF_MODULEDIR OUTPUT_STRIP_TRAILING_WHITESPACE)
 install(TARGETS pixbufloader-jxl DESTINATION "${GDK_PIXBUF_MODULEDIR}")
